cdk/package.json

```diff
@@ -11,7 +11,7 @@
     "diff": "cdk diff --path-metadata false --version-reporting false"
   },
   "devDependencies": {
-    "@guardian/cdk": "43.0.0",
+    "@guardian/cdk": "43.3.1-beta.1",
```

Comment:
This pattern is very new and I needed to pick up some fixes, hence the beta version

Reply:
As mentioned in https://github.com/guardian/mobile-save-for-later/pull/64#discussion_r880456331, we should move to the latest released version before merging.